lib/rules/button-has-type.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
   meta: {
     docs: {
       description: 'Forbid "button" element without an explicit "type" attribute',
-      category: 'Possible Errors',
+      category: 'Best Practices',
```

Comment:
this is definitely a possible error; button elements must never omit a "type" attribute